coralproject/talk
# [next] Responsive Support

## What does this PR do?
- Implement `MatchMedia`, `Flex` Component
- Use CSS variables which support dynamically changing values inside `media queries`
- Keep Sass style variables for defining media queries e.g. `@media (min-width: $breakpoints-xs) {`
- Use a default spacing unit of `5` in small screens and `10` otherwise. 
- Remove `precss` preset in favor of our own selection of plugins that we actually use.

## How do I test this PR?

- `npm run docz:watch` to check out the new components.
- `npm run watch` and decrease / increase screen size to see dynamic spacing unit in practise.
